src/main/haskell/kore/src/Kore/Step/SmtLemma.hs

```diff
@@ -143,12 +146,15 @@ declareSMTLemmas m = SMT.liftSMT $ do
         :: Given (MetadataTools Object StepperAttributes)
         => Sort Object
         -> MaybeT SMT SExpr
-    translateSort sort@(SortActualSort (SortActual _ children)) =
-        case getSmtlib $ smtlib $ sortAttributes given sort of
-            Just sExpr -> do
-                children' <- mapM translateSort children
-                pure $ applySExpr sExpr children'
-            Nothing -> mzero
+    translateSort sort@(SortActualSort (SortActual (getId -> name) children))
+        | name == "SortInt" = return (SMT.Atom "Int")
+        | name == "SortBool" = return (SMT.Atom "SortBool")
```

Comment:
Shouldn't this look at the `hook` attribute? Should this be `Bool` instead of `SortBool`?

Reply:
I had a discussion with Dwight about this. 

We don't want to make it dependent on the `smtlib` attribute because in the future this might force users of K to write the attribute everywhere.

So, I decided to hardcode the case for `Int` and `Bool`. I just hardcoded it based on their names in Kore. Maybe using hook is better....